models.py

```diff
@@ -1401,3 +1403,36 @@ def __init__(self, *args, **kwargs):
 			nonlinearity = ('leaky_relu', 0.01),
 			**kwargs
 		)
+
+
+class OnnxModelWraper(nn.Module):
+	def __init__(self, onnx_model_path, provider=None):
```

Comment:
Please take in logging severity as an argument, just like done in `train.py`

Reply:
Please give me a link to train.py example, I can't find example.

Reply:
https://github.com/vadimkantorov/convasr/search?q=severity